### PR TITLE
Update version and improve downloads on archived datasets

### PR DESCRIPTION
**Pull request by henrykironde**
Solve backward compatibility caused by updated  feature 
download from archives

### CODE REVIEW

File: scripts/gentry_forest_transects.py [hunk 1]
```diff
@@ -62,7 +62,8 @@ def download(self, engine=None, debug=False):
         local_zip.close()
         self.engine.download_files_from_archive(self.urls["stems"], filelist)
 
-        filelist = [os.path.basename(filename) for filename in filelist]
+        if parse_version(VERSION).__str__() < parse_version("2.1.dev").__str__():
+            filelist = [os.path.basename(filename) for filename in filelist]
```
Comment (henrykironde) [inline scripts/gentry_forest_transects.py]: For older versions, we need to use the file name,  not its relative path.

**Comment by henrykironde** [comment]
@ethanwhite, I think this will solve the nd-gain dataset installation errors

**Comment by henrykironde** [comment]
This is weird that it is now failing.  Let me check it again

**Comment by ethanwhite** [comment]
Looks great @henrykironde! Sorry for the delay in getting to the review.

**Pull request merged**